core/src/main/java/org/apache/accumulo/core/client/impl/MasterClient.java

```diff
@@ -44,14 +44,14 @@
   public static MasterClientService.Client getConnectionWithRetry(ClientContext context) {
     while (true) {
 
-      MasterClientService.Client result = getConnection(context);
+      MasterClientService.Client result = getConnection(context, false);
```

Comment:
In order to limit the number of changes, it's better to keep a method with the original method signature that wraps the new version with the extra parameter, so this (and other callers which are expected to keep their original behavior) is unaffected by this change.

Example:

```java
public static MasterClientService.Client getConnection(ClientContext context) {
  return getConnection(context, false); // or getConnection(context, 0) with my suggestion to specify the timeout
}
```

Reply:
ok, I have done it the other way around. Will check that.